ocs_ci/framework/conf/default_config.yaml

```diff
@@ -82,12 +82,22 @@ ENV_DATA:
   worker_replicas: 3
   skip_ocp_deployment: false
   skip_ocs_deployment: false
+  # --------------------------------------------------------------------------
+  # Following parameters are not used when deploy via operator and will removed
+  # soon as part of cleanup: https://projects.engineering.redhat.com/browse/OCSQE-73
   # Do not change to specific version like v14.2.1-20190430 if not needed
   # cause we don't need to update it each time new 14.x version is released
   # but only once when move to new version like v15.
-  ceph_image: 'ceph/ceph:v14.2.3-20190904'
-  rook_image: 'rook/ceph:master'
+  ceph_image: 'quay.io/rhceph-dev/rhceph-4.0-rhel-8:latest'
+  rook_image: 'quay.io/rhceph-dev/rook:latest'
+  # DS CSI sidecars images mentioned in https://jira.coreos.com/browse/RHSTOR-150
+  rook_csi_registrar_image: 'quay.io/openshift/origin-csi-node-driver-registrar:4.2'
+  rook_csi_provisioner_image: 'quay.io/openshift/origin-csi-external-provisioner:4.2'
+  rook_csi_snapshotter_image: 'quay.io/openshift/origin-csi-external-snapshotter:4.2'
+  rook_csi_attacher_image: 'quay.io/openshift/origin-csi-external-attacher:4.2'
+  # End of the unused parameters for operator deployment
+  # --------------------------------------------------------------------------
```

Comment:
One line below.
Should be m5.4xlarge?

Reply:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/ocsci/production-aws-ipi.yaml
For production jobs we are using this config. We can have one general production config which will not be bounded to AWS-IPI later on but we don't want to all ppl to have this bigger instance type as default IIRC, just for production jobs as we don't have enough resources to allow everyone to deploy on m5.4xlarge